### PR TITLE
feat: add system cell crate

### PR DESCRIPTION
**Pull request by driftluo**
Move `system_cells`l to this repository

Subsequent removal of binary from git(May lead to misuse, need to consider better practices)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[workspace]
+members = ["system-cells", "tests"]
```
Comment (zhangsoledad) [inline Cargo.toml]: feed new line

**Comment by xxuejie** [comment]
Can we exclude the binary here? What about use a separate script to trigger the build, and fault `cargo build` when that binary is not present, or the binary hash does not match?

File: system-cells/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[package]
+name = "system-cells"
```
Comment (doitian) [inline system-cells/Cargo.toml]: Name it `ckb-system-cells`

Comment (doitian) [reply system-cells/Cargo.toml]: I think we don't need to keep the two crates. Just use the top crate for the lib and use the name `ckb-system-scripts`, and use cargo tests directory to test the scripts.

**Pull request merged**